src/components/Carousel/Card.css

```diff
@@ -0,0 +1,34 @@
+.card-body {
+    position: absolute;
+    top: 39%;
+    width: 17.9rem;
+    height: 11rem;
+    background-color: beige;
+}
+
+.card {
+    width: 18rem;
+    color: black;
+}
+
+.icon {
+    width: 3rem;
+}
+
+.filter {
+    max-width: 40%;
+}
+
+/* .css-1qzevvg {
```

Comment:
can we remove this comment, and rename the class selector to something more readable?